14/main.js

```diff
@@ -0,0 +1,68 @@
+const centuryInputField = document.getElementById("task1");
+const vowelOneInputField = document.getElementById("task2");
+const spinWordsInputField = document.getElementById("task3");
+const highAndLowInputField = document.getElementById("task4");
+const createPhoneNumberInputField = document.getElementById("task5");
+const centuryButton = document.getElementById("task1_button");
+const vowelOneButton = document.getElementById("task2_button");
+const spinWordsButton = document.getElementById("task3_button");
+const highAndLowButton = document.getElementById("task4_button");
+const createPhoneNumberButton = document.getElementById("task5_button");
+function whatCentury(year) {
```

Comment:
Nice to have a more meaningful name, for example, `getCentury`
Also, it works with strings, empty values and negative numbers
<img width="181" alt="image" src="https://user-images.githubusercontent.com/84010501/200787680-15c505a0-5307-4578-970b-c6819badf627.png">

<img width="181" alt="image" src="https://user-images.githubusercontent.com/84010501/200788980-8796d137-44f8-42a7-8191-3d4a1b2a5d3e.png">